src/firestore/util.ts

```diff
@@ -173,10 +220,11 @@ export const initializeFirebaseProject = async (
     }
     const kit = {
       firebaseApp: app,
-      auth: optionallyMarkRaw('auth', getAuth(app)),
-      db: optionallyMarkRaw('db', getFirestore(app)),
-      functions: optionallyMarkRaw('functions', getFunctions(app)),
-      storage: optionallyMarkRaw('storage', getStorage(app)),
+      appCheckToken: appCheckToken.token,
```

Comment:
Same here, if the `siteKey` and `debugToken` were undefined when passed in, we should return `null` or something.